.gitignore

```diff
@@ -1,2 +1,9 @@
 
 /gtfs-netex-test/aux_test_input/sncf-test
+gtfs-netex-test/PROD_NETEX_TT_1.10_CHE_SKI_2024_OEV-SCHWEIZ_COMMON_1_1_202406270606.xml
+gtfs-netex-test/PROD_NETEX_TT_1.10_CHE_SKI_2024_OEV-SCHWEIZ_RESOURCE_1_1_202406270606.xml
+gtfs-netex-test/LINE-F.xml
+gtfs-netex-test/aux_test_input/swiss.zip
+gtfs-netex-test/aux_test_input/solea.zip
+gtfs-netex-test/aux_test_input/milano.zip
+/gtfs-netex-test/aux_test_input
```

Comment:
The last and first path are obviously wrong (they are not relative).
But I would also say that this does not belong here, ignore a folder, fine, but not instances of files.

Reply:
I will change this one. No more zips. Only a very tiny example folder of such things.